src/sass/base/_mixins.sass

```
@@ -0,0 +1,63 @@
+@mixin flex
+    display: flex
+    align-items: center
+    justify-content: center
+
+@mixin screen ($size)
+    @media screen and (min-width: $size)
+        @content
+
+@mixin pt-pb($top, $bottom)
+    padding-top: $top
+    padding-bottom: $bottom
+
+@mixin pl-pr($left, $right)
+    padding-left: $left
+    padding-right: $right
+
+@mixin mt-mb($top, $bottom)
+    margin-top: $top
+    margin-bottom: $bottom
+
+@mixin ml-mr($left, $right)
+    margin-left: $left
+    margin-right: $right
+
+@mixin title 
+    color: $title-color
+    font-size: 1.2rem
+    font-weight: $medium
+    text-transform: uppercase
+
+@mixin text
+    color: $bold-gray
+    font-family: $secondary-font
+
+@mixin transition
+    transition: all .3s ease-in-out
+    -moz-transition: all .3s ease-in-out
+    -webkit-transition: all .3s ease-in-out
+    -o-transition: all .3s ease-in-out
+    -ms-transition: all .3s ease-in-out
+
+@mixin cart
+    top: 0
+    right: 0
+    position: absolute
+    &:before
+        content: "0"
```

Có thể sử dụng data attribute để thay thế
```suggestion
        content: attr(data-total-cart)
```